src/events.rs

```diff
@@ -0,0 +1,28 @@
+// This file is Copyright its original authors, visible in version control
+// history.
+//
+// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
+// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// You may not use this file except in accordance with one or both of these
+// licenses.
+
+//! Events are returned from various bits in the library which indicate some action must be taken
+//! by the client.
+//!
+//! Because we don't have a built-in runtime, it's up to the client to call events at a time in the
+//! future.
+use bitcoin::secp256k1::PublicKey;
+
+/// An Event which you should probably take some action in response to.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum Event {
+	/// A response to a ListProtocols request that informs the caller
```

Comment:
yeah I guess currently there's really no reason to use it.  I guess the idea is in a world where your wallet lets you add arbitrary LSPs it could use this to surface which ones support JIT Channels or whatever feature you are interested in using.

you could also just try to issue the protocol request and you'll get an error back without ever using list protocols.

should I just remove it for now?